### ISSUE TITLE
How to visulaise the  output

### ISSUE THREAD

**Issue by Adil42s** [nota-github/AIC2024_Track1_Nota#6]
Thank you for your wonderful contribution.Could you provide detailed instructions or examples on how to visualize the output?.I tried  but the videos are corrupted . thank you

**Comment by qkqk772** [comment]
Does it still not save the video properly when you set 'write_vid' to True (line 129) in 'eval_mcpt.py'?

**Comment by Adil42s** [comment]
> Does it still not save the video properly when you set 'write_vid' to True (line 129) in 'eval_mcpt.py'?

Yes,it solved,thank you .which score you are  computing along with the tracking ID, it is bounding box confidence score

**Comment by qkqk772** [comment]
Yes, it is bbox confidence score :)

**Comment by wyhdata97** [comment]
why output videos are corrupted? do you have some examples?

**Comment by OnceUponATimeMathley** [comment]
> Thank you for your wonderful contribution.Could you provide detailed instructions or examples on how to visualize the output?.I tried but the videos are corrupted . thank you

Could you please share the password for this dataset with me? Thank you.
I tried requesting it but have not received a response from the NVIDIA AI Challenge team.
I would like to reproduce the results from one of the top teams.
If possible, please share it via my email: longchauhl@gmail.com.
Thanks again!
